examples/lang/flipper/src/lib.rs

```diff
@@ -17,13 +17,15 @@
 #![cfg_attr(not(any(test, feature = "test-env")), no_std)]
 
 use ink_core::{
-    env::println,
+    env::DefaultSrmlTypes,
     memory::format,
     storage,
 };
 use ink_lang::contract;
 
 contract! {
+    type EnvTypes = DefaultSrmlTypes;
```

Comment:
Maybe we want to use another syntax here that is less confused with an actual type alias which this isn't behind the scenes.
This is more of a meta information to my understanding.
So maybe we could make use of some attribute-like syntax here.
```rust
#![env = DefaultSrmlTypes]
```
Or something like that.

Reply:
Awaiting outcome of poll on channel